contracts/@openzeppelin/contracts-ethereum-package/contracts/GSN/Context.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
+
+import "../../../upgrades/contracts/Initializable.sol";
+
+/*
+ * @dev Provides information about the current execution context, including the
+ * sender of the transaction and its data. While these are generally available
+ * via msg.sender and msg.data, they should not be accessed in such a direct
+ * manner, since when dealing with GSN meta-transactions the account sending and
+ * paying for execution may not be the actual sender (as far as an application
+ * is concerned).
+ *
+ * This contract is only required for intermediate, library-like contracts.
+ */
+contract Context is Initializable {
```

Comment:
Is this so support the GSN?

Reply:
Yeah the OZ stuff tries to support it... not sure it matters in this case though.